### PR TITLE
Switch to npm7 and use latests global configs

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

**Comment by marcelklehr** [comment]
Thanks :heart: I really appreciate it!
JS tooling, even though it's great when it's working, is a bit of a nightmare for me :D

**Comment by marcelklehr** [comment]
Stylelint fails, though.

**Review by marcelklehr — APPROVED** [review]
:heart: :rocket:

File: package.json [hunk 1]
```diff
@@ -45,47 +45,16 @@
...
-		"stylelint": "^13.12.0",
-		"stylelint-config-recommended-scss": "^4.2.0",
-		"stylelint-scss": "^3.19.0",
-		"stylelint-webpack-plugin": "^2.1.1",
-		"url-loader": "^4.1.1",
-		"vue-loader": "^15.9.6",
-		"vue-template-compiler": "^2.6.12",
-		"webpack": "^5.4.0",
-		"webpack-cli": "^4.5.0",
-		"webpack-dev-server": "^3.11.2"
```
Comment (skjnldsv) [inline package.json:85]: @marcelklehr should be easier now as all global configs inherit dependencies :)
Less worrying (hopefully)

File: src/components/Bookmark.vue [hunk 2]
```diff
@@ -206,6 +206,7 @@ export default {
 			this.renaming = true
 		},
 		async onRenameSubmit(title) {
+			// eslint-disable-next-line vue/no-mutating-props
```
Comment (skjnldsv) [inline src/components/Bookmark.vue:209]: Bad @marcelklehr ! :grin: 
Please fix it in the future, this is not how you're supposed to update props in vue :wink:

Comment (marcelklehr) [reply src/components/Bookmark.vue:209]: Caught me :smile:

**Pull request merged**